modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -1,12 +1,40 @@
 = Guide to Named Examples in RAML 1.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-If you want to use one or more named examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these three rules:
+If you want to use examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these rules:
```

Comment:
- `If you want to use examples...` -> `When defining examples...`
- we're talking about external/included examples here, so I would mention it -> `When defining examples in external files...`
- `must` is a little too prescriptive here IMO. I would either use `should` instead or rephrase a bit: `(...) , you can follow these simple rules:`